### PR TITLE
docs(datepicker): decompose Basic demo (remove presetting dates for d…

### PR DESCRIPTION
**Pull request by ghost**
1.remove presetting dates for datepicker and daterangepicker (basic section) http://joxi.ru/4AkgoOwtMpRdeA
2. add new section : Initial state of a date, which set using bsValue, [(ngModel)] http://joxi.ru/Vm6ZxbeHxl7wqm
related to #3669

### CODE REVIEW

File: demo/src/app/components/+datepicker/demos/date-initial-state/date-initial-state.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'demo-datepicker-date-initial-state',
+  templateUrl: './date-initial-state.html'
+})
+export class DemoDatepickerDateInitialStateComponent {
+  bsValue: Date = new Date();
+  bsRangeValue: any = [new Date(2017, 7, 4), new Date(2017, 7, 20)];
```
Comment (IlyaSurmay) [inline demo/src/app/components/+datepicker/demos/date-initial-state/date-initial-state.ts]: Change the range to a week from today. you can do this by example from min-max demo

File: demo/src/app/components/+datepicker/datepicker-section.list.ts [hunk 2]
```diff
@@ -51,7 +51,14 @@ export const demoComponentContent: ContentSection[] = [
             </li>
           </ul>
         `,
-        outlet: DemoDatePickerPopupComponent
+        outlet: DemoDatepickerBasicComponent
+      },
+      {
+        title: 'Initial state of a date',
```
Comment (IlyaSurmay) [inline demo/src/app/components/+datepicker/datepicker-section.list.ts]: just `Initial state`

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]

**Review by IlyaSurmay — DISMISSED** [review]

**Comment by ghost** [comment]
update and testing my branch according to Yevheniia comments(conflicts in files) which I can't see

File: demo/src/app/components/+datepicker/demos/date-initial-state/date-initial-state.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'demo-datepicker-date-initial-state',
+  templateUrl: './date-initial-state.html'
+})
+export class DemoDatepickerDateInitialStateComponent {
+  bsValue: Date = new Date();
```
Comment (IlyaSurmay) [inline demo/src/app/components/+datepicker/demos/date-initial-state/date-initial-state.ts]: remove type annotation
if type annotation isn't provided, typescript will use the type of the first assigned value

File: demo/src/app/components/+datepicker/demos/date-initial-state/date-initial-state.ts [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  bsRangeValue: Date[];
+  maxDate: Date = new Date();
```
Comment (IlyaSurmay) [inline demo/src/app/components/+datepicker/demos/date-initial-state/date-initial-state.ts]: same

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]

**Review by IlyaSurmay — DISMISSED** [review]

**Comment by YevheniiaMazur** [comment]
Tested, looks good

**Review by valorkin — DISMISSED** [review]

**Pull request merged**